src/commands/new.ts

```diff
@@ -552,13 +553,48 @@ export default {
     }
     // #endregion
 
+    // #region Format generator templates EOL for Windows
+    let warnAboutEOL = false
+    if (isWindows) {
+      // find unix2dos to help convert EOL, usually installed with git, ie:  C:\Program Files\Git\usr\bin\unix2dos.EXE
+      const unixToDosPath = await system.exec("which unix2dos")
+      // find all templates Ignite provides by default
+      const templates = filesystem.find(`${targetPath}/ignite/templates`, {
+        directories: false,
+        files: true,
+        matching: "*.ts*.ejs",
```

Comment:
I think we are going to want to use any `.ejs` file here since we can also do non-ts files. 
```suggestion
        matching: "*.ejs",
```